src/core/ext/transport/chttp2/server/chttp2_server.cc

```diff
@@ -596,6 +596,7 @@ grpc_error_handle Chttp2ServerListener::Create(
   // The bulk of this method is inside of a lambda to make cleanup
   // easier without using goto.
   grpc_error_handle error = [&]() {
+    grpc_error_handle error = GRPC_ERROR_NONE;
```

Comment:
how was this working earlier?

Reply:
Previously outer `error` was captured and used so it worked in a strange way but it somehow worked. I guess It's mainly acceptable because it's a c pointer. With a C++ instance, however, it becomes more strict and it stops working in this way.